WordPress/src/main/java/org/wordpress/android/widgets/PostListButton.java

```diff
@@ -81,7 +86,16 @@ public void setButtonType(int buttonType) {
 
         mButtonType = buttonType;
         mTextView.setText(getButtonTextResId(buttonType));
-        mImageView.setImageResource(getButtonIconResId(buttonType));
+        if (buttonType == BUTTON_RETRY) {
```

Comment:
Thank you for pointing this out and creating the XML drawable for me! 🙇 addressed in e92a672